source/manual/create-a-gpg-key.html.md

```diff
@@ -4,10 +4,12 @@ title: Create a GPG key
 parent: "/manual.html"
 layout: manual_layout
 section: Support
-last_reviewed_on: 2017-04-01
+last_reviewed_on: 2017-09-06
 review_in: 6 months
 ---
 
+We use GPG keys to encryt our sercets, details for getting access to with a GPG key can be found [here](/manual/encrypted-hiera-data.html#common-tasks-for-handling-encrypted-hiera-data).
```

Comment:
Couple of spelling corrections: "encrypt" and "secrets"

Reply:
Also, I think a word is missing here: "getting access to ? with a GPG key can be found"

Reply:
reworded to clarify what the link is for

Reply:
reworded to clarify what the link is for